components/script/dom/webidls/Location.webidl

```diff
@@ -6,7 +6,7 @@
 // https://html.spec.whatwg.org/multipage/#location
 [Unforgeable] interface Location {
   /*stringifier*/ attribute USVString href;
-  //         attribute USVString origin;
+           readonly attribute USVString origin;
```

Comment:
The spec actually has the `attribute` bit line up here with subsequent lines, and we try to keep our WebIDL snippets as close to the spec as possible.